CMakeLists.txt

```diff
@@ -99,6 +96,10 @@ else()
 endif()
 
 
+# Create versions.hpp and add binary dir to include dirs
+configure_file("${PROJECT_SOURCE_DIR}/src/versions.hpp.in" "${PROJECT_BINARY_DIR}/versions.hpp")
```

Comment:
Please restrict substitution using `@ONLY` to enforce consistency in the future.

https://cmake.org/cmake/help/latest/command/configure_file.html